README.md

```diff
@@ -233,6 +233,20 @@ Same as `sanitize(fields)`, but only sanitizing `req.query`.
 
 ---
 
+## Wildcards (*)
+To validate or sanitize an array of strings use wildcards(*).
```

Comment:
This sentence is not necessarily the only case to use wildcards, so mentioning something like "nested fields" would be better.

Reply:
Cool.